docs/src/pages/quasar-cli-vite/state-management-with-pinia.md

```diff
@@ -148,3 +148,21 @@ export default {
 ```
 
 [More info on defining a Pinia store](https://pinia.vuejs.org/core-concepts/).
+
+## Accessing the router in Pinia stores
+
+Simply use `this.router` in Pinia stores to get access to the router.
+
+See an example of accessing router in Pinia store action:
```

Comment:
I'd say "Here is an example" here or just remove this line as is on the other pages.